src/stories/components/404/not-found-component-404.tsx

```diff
@@ -182,7 +182,7 @@ const TextUps = styled(Typography)<{ isLight?: boolean }>(({ isLight }) => ({
   },
 }));
 
-const TextDescription = styled(Typography)({
+const TextDescription = styled(Typography, { shouldForwardProp: (prop) => prop !== 'isLight' })({
```

Comment:
I am in doubt relate with one, why should use shouldForwadProps if we are not going to use is Light props

Reply:
Solved